engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-manage-packages.html

```diff
@@ -25,8 +25,13 @@ <h3 translate>Packages in {{ repository.name }}</h3>
   </div>
 
   <div data-block="actions">
+    <div bst-modal="removeContent()">
+      <div data-block="modal-header" translate-plural>Remove {{ detailsTable.numSelected  }} Packages?</div>
+      <div data-block="modal-body" translate-plural>Are you sure you want to remove the {{ detailsTable.numSelected }} packages selected?</div>
```

Comment:
No, it's not.

You need to provide `translate-n` so it knows whether or not to use the plural and `translate-plural' should have the plural message, see [the docs](https://angular-gettext.rocketeer.be/).